From fdb9f4a4fe55c28d18cca43a142933e2a8ab18ec Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Mon, 10 Sep 2018 16:43:24 +0300
Subject: [PATCH 314/706] xgmac-mdio: remove a bogus iounmap

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
(cherry picked from commit 36676b7b0421b0a03ab07377c2f6dae0fe9587f8)
(cherry picked from commit 0c2a36ef1081b37b7f401cf7161a3de62f69b2c0)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/xgmac_mdio.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/xgmac_mdio.c b/drivers/net/ethernet/freescale/xgmac_mdio.c
index f9e4db007718..5a160adcf81d 100644
--- a/drivers/net/ethernet/freescale/xgmac_mdio.c
+++ b/drivers/net/ethernet/freescale/xgmac_mdio.c
@@ -345,7 +345,6 @@ static int xgmac_mdio_remove(struct platform_device *pdev)
 
 	mdiobus_unregister(bus);
 	iounmap(priv->map);
-	iounmap(bus->priv);
 	mdiobus_free(bus);
 
 	return 0;
-- 
2.17.1

